### PR TITLE
py-macs2: New package

### PR DESCRIPTION
**Pull request by johnwfinigan**
Genomics software. This is my first PR, all feedback welcome.

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-macs2/package.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    version('2.1.1.20160309', '2008ba838f83f34f8e0fddefe2a3a0159f4a740707c68058f815b31ddad53d26')
+
+    extends('python')
+    depends_on('python@2.7:2.8')
+
+    # Most Python packages only require py-setuptools as a build dependency.
+    # However, py-macs2 requires py-setuptools during runtime as well.
+    depends_on('py-setuptools', type=nolink)
+    depends_on('py-numpy', type=nolink)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-macs2/package.py]: According to the `setup.py`, it specifically needs `'numpy>=1.6'`, so I would use:
```python
depends_on('py-numpy@1.6:', type=nolink)
```

**Comment by citibeth** [comment]
After @adamjstewart's changes, LGTM

File: var/spack/repos/builtin/packages/py-macs2/package.py (hunk 1 above)
Comment (johnwfinigan) [inline var/spack/repos/builtin/packages/py-macs2/package.py]: Thank you for the review!

**Comment by alalazo** [comment]
Is there any reason why this package has been called `macs2` instead of `macs`? I have a user asking for version `1.4.1`, which comes from the same repository, so I was wondering if a change in the name of the package was fine with everybody.

**Comment by adamjstewart** [comment]
When creating Python packages, I always go with the name on PyPI. If you want to `spack install` a package, it should have the same name as you would use for `pip install`, but with a `py-` prefix.

It looks like there is a PyPI package for both MACS and MACS2. If the developers decided to change their package name on PyPI, they must have had a good reason. I would respect their wishes and create a second package.

**Pull request merged**